components/ruby/ruby-23/ruby-23-tk.p5m

```diff
@@ -0,0 +1,367 @@
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/gif.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/ico.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/jpeg.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/pcx.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/pixmap.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/png.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/ppm.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/ps.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/raw.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/setup.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/sgi.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/sun.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/tga.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/tiff.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/window.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/xbm.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tkimg/xpm.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tktable.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tktable/setup.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tktable/tktable.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tktrans.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tktrans/setup.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/tktrans/tktrans.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/treectrl.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/treectrl/setup.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/treectrl/tktreectrl.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/trofs.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/trofs/setup.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/trofs/trofs.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/version.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/vu.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/vu/bargraph.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/vu/charts.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/vu/dial.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/vu/pie.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/vu/setup.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/vu/spinbox.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/winico.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/winico/setup.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkextlib/winico/winico.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkfont.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkmacpkg.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkmenubar.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkmngfocus.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkpalette.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkscrollbox.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tktext.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkvirtevent.rb
+file path=usr/ruby/2.3/lib/ruby/2.3.0/tkwinpkg.rb
+license COPYING license="Ruby License"
```

Comment:
as for ruby-23.p5m, 
a) fix copyright
b) we don't have to depend on ruby 18 anymore as it's marked obsolete